lasagne/tests/layers/test_conv.py

```diff
@@ -287,7 +284,7 @@ def test_init_none_nonlinearity_bias(self, Conv2DImpl, DummyInputLayer):
         assert layer.b is None
 
     def test_invalid_pad(self, Conv2DImpl, DummyInputLayer):
-        input_layer = DummyInputLayer((1, 2, 3))
+        input_layer = DummyInputLayer((1, 2, 3, 3))
```

Comment:
Wait, how did this pass before if the input layer only had 3 axes? Or is it because it was a `DummyInputLayer`?